plugins/modules/net_tools/nmcli.py

```diff
@@ -100,7 +100,8 @@
     routing_rules4:
         description:
             - Is the same as in an C(ip route add) command, except always requires specifying a priority.
-        type: str
+        type: list
+        element: str
```

Comment:
Was it possible to have a comma in this option's value when it was a string?

Reply:
I can check `nmcli` source, but basic testing indicates `,` can only appear at the beginning or end of a rule to separate them. Otherwise the actual rule string is expected to be space delimited and consist of the keywords `priority`, `from`, and `table` with corresponding values.

```
nmcli connection add type ethernet con-name Internal-Workstations ifname enp8s0 ipv4.method manual ipv4.addresses 10.0.0.1/24 ipv4.routes "10.0.0.0/24 src=192.0.2.1 table=5000" ipv4.routing-rules "priority, 5 from 10.0.0.0/24 table 5000" connection.zone internal
Error: failed to modify ipv4.routing-rules: missing argument for "priority".
nmcli connection add type ethernet con-name Internal-Workstations ifname enp8s0 ipv4.method manual ipv4.addresses 10.0.0.1/24 ipv4.routes "10.0.0.0/24 src=192.0.2.1 table=5000" ipv4.routing-rules "priority 5, from 10.0.0.0/24 table 5000" connection.zone internal
Error: failed to modify ipv4.routing-rules: rule is invalid: invalid priority.
nmcli connection add type ethernet con-name Internal-Workstations ifname enp8s0 ipv4.method manual ipv4.addresses 10.0.0.1/24 ipv4.routes "10.0.0.0/24 src=192.0.2.1 table=5000" ipv4.routing-rules "priority 5 from ,10.0.0.0/24 table 5000" connection.zone internal
Error: failed to modify ipv4.routing-rules: missing argument for "from".
nmcli connection add type ethernet con-name Internal-Workstations ifname enp8s0 ipv4.method manual ipv4.addresses 10.0.0.1/24 ipv4.routes "10.0.0.0/24 src=192.0.2.1 table=5000" ipv4.routing-rules "priority 5 from 10.0.0.0/24, table 5000" connection.zone internal
Error: failed to modify ipv4.routing-rules: rule is invalid: invalid priority.
nmcli connection add type ethernet con-name Internal-Workstations ifname enp8s0 ipv4.method manual ipv4.addresses 10.0.0.1/24 ipv4.routes "10.0.0.0/24 src=192.0.2.1 table=5000" ipv4.routing-rules "priority 5 from 10.0.0.0/24 table, 5000" connection.zone internal
Error: failed to modify ipv4.routing-rules: missing argument for "table".
```